tests/test_prompt_creation.py

```diff
@@ -61,9 +61,10 @@ def test_final_prompt_generation(self):
         final_prompt = self.prompt_template.get_prompt_from_messages(
             self.test_case["messages"], tools_or_functions
         )
+
         self.assertEqual(
-            final_prompt,
-            self.final_prompt,
+            final_prompt.strip(),
```

Comment:
Why this was necessary? What is the output of this if we dont add strip

Reply:
final_prompt return: "\n" at the end of the prompt

Reply:
You can see in the implementation: https://github.com/MeetKai/functionary/blob/main/functionary/prompt_template/prompt_template_v2.py#L80
at the end of each message is: "\n"